docs/DEVELOPER_GUIDE.md

```diff
@@ -333,5 +333,12 @@ Ideally, the `version` specified should match the release tag of the plugin.
 This helps users and maintainers to easily identify which version of the plugin
 they have installed.
 
+### Automating the releasing of new version
+
+You can use Github Actions to publish new release of your Krew plugin. 
+`krew-release-bot` is a github action to automatically bump the version in 
```

Comment:
```suggestion
`krew-release-bot` is a GitHub Action to automatically bump the version in 
```
Better be consistent with regards to capitalization. IMO, `Action` is a name here and needs to have a capital letter too.

Reply:
done